### PR TITLE
 minor touchups

### PR DESCRIPTION
**Pull request by keepforever**
Fix typo in `Input.jsx` and add missing `key` values.

### CODE REVIEW

File: src/pages/Utils.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const UtilsPage = () => (
             <div className="bd-example">
                 {brandColors.map(color => {
                     return (
-                        <p className={classnames('p-3 mb-2', `bg-${color}`, {
+                        <p key={`txt-${color}`} className={classnames('p-3 mb-2', `bg-${color}`, {
```
Comment (cwardzala) [inline src/pages/Utils.jsx]: I think this should be `bg-${color}`

File: src/pages/Content.jsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ const ContentPage = () => {
                 <div className="swatch-list">
                     <div className="row">
                         {brandColors.map(color => (
-                            <div className="col-xs-6 col-md-2 txt-center">
+                            <div key={`txt-${color}`} className="col-xs-6 col-md-2 txt-center">
```
Comment (cwardzala) [inline src/pages/Content.jsx]: this should probably be `brand-${color}`

File: src/pages/Content.jsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ const ContentPage = () => {
                 <div className="swatch-list mb-3">
                     <div className="row">
                         {colors.map(color => (
-                            <div className="col-xs-6 col-md-2 txt-center">
+                            <div key={`txt-${color}`} className="col-xs-6 col-md-2 txt-center">
```
Comment (cwardzala) [inline src/pages/Content.jsx]: should make this key `colors-${color}`.

File: src/pages/Components.jsx [hunk 4]
```diff
@@ -32,6 +32,7 @@ const ComponentsPage = () => {
                 <p>
                     {brandColors.map(color => (
                         <button
+                            key={`txt-${color}`}
```
Comment (cwardzala) [inline src/pages/Components.jsx]: this key should be `btn-${color}`.

File: src/pages/Components.jsx [hunk 5]
```diff
@@ -42,6 +43,7 @@ const ComponentsPage = () => {
...
```
Comment (cwardzala) [inline src/pages/Components.jsx]: this key should be `btn-outline-${color}`.

File: src/pages/Utils.jsx (hunk 1 above)
Comment (keepforever) [reply src/pages/Utils.jsx]: ✅

File: src/pages/Content.jsx (hunk 2 above)
Comment (keepforever) [reply src/pages/Content.jsx]: ✅

File: src/pages/Content.jsx (hunk 3 above)
Comment (keepforever) [reply src/pages/Content.jsx]: ✅

File: src/pages/Components.jsx (hunk 5 above)
Comment (keepforever) [reply src/pages/Components.jsx]: ✅

File: src/pages/Components.jsx (hunk 4 above)
Comment (keepforever) [reply src/pages/Components.jsx]: ✅

**Pull request merged**